lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -46,14 +53,18 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     % if share_settings:
                       <%
                        facebook_share_text = _("I completed the {course_title} course on {platform_name}.").format(course_title=accomplishment_course_title, platform_name=platform_name)
+                       twitter_share_text = _("I completed the {course_title} course on {platform_name} and here is the link to my certificate.").format(course_title=accomplishment_course_title, platform_name=platform_name)
```

Comment:
I'm not sure what the outcome was from various discussions. Is platform name potentially an issue too and is it OK to include a URL?
How about something like this: 
"I completed a course on {platform name}. See my certificate {here}!" 
@pbaruah @mattdrayer @asadiqbal08 @marcotuts - thoughts?